src/storage/pglite-adapter.ts

```
@@ -39,6 +50,125 @@ import {
+      ...(results.affectedRows
+        ? { numAffectedRows: BigInt(results.affectedRows) }
+        : {}),
+    };
+  }
+
+  public async *streamQuery<R>(
+    _compiledQuery: CompiledQuery,
+    _chunkSize?: number
+  ): AsyncIterableIterator<QueryResult<R>> {
+    throw new Error(`PGliteDriver doesn't support streaming.`);
+  }
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)